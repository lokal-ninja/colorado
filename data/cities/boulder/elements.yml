id: 1305391500
type: node
latitude: 40.0244
longitude: -105.25512
postcode: "80301"
city: Boulder
street: "30th Street"
housenumber: 2321
phone: ""
opening_hours: "Mo-Su 09:00-21:00"
website: "https://elementsmassage.com/boulder/"
