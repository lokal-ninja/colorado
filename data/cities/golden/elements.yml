id: 9432919178
type: node
latitude: 39.74956
longitude: -105.22329
postcode: "80401"
city: Golden
street: 16th Street
housenumber: "1200"
phone: ""
opening_hours: ""
website: ""
